x-pack/plugins/actions/common/connector_feature_config.ts

```diff
@@ -72,11 +80,18 @@ export const SecuritySolutionFeature: ConnectorFeatureConfig = {
   compatibility: compatibilityAlertingRules,
 };
 
+export const GeneralFeature: ConnectorFeatureConfig = {
+  id: GeneralConnectorFeatureId,
+  name: compatibilityGeneral,
+  compatibility: compatibilityGeneral,
+};
+
 const AllAvailableConnectorFeatures = {
   [AlertingConnectorFeature.id]: AlertingConnectorFeature,
   [CasesConnectorFeature.id]: CasesConnectorFeature,
   [UptimeConnectorFeature.id]: UptimeConnectorFeature,
   [SecuritySolutionFeature.id]: SecuritySolutionFeature,
+  [GeneralFeature.id]: GeneralFeature,
```

Comment:
@mikecote @ymao1 What do you think about this new category? I found it a bit confusing. As the connector will be used in security solution do we need this?

Reply:
It might be best to put it under `SecuritySolutionFeature`, however I can see that they may not want it to show as an available rule action 🤔 ? If so, we may need to create one for "security solution rule actions" and another for wherever this connector becomes available in security solution (ex: "security solution manual actions").

Reply:
I believe o11y has plans to utilize this connector as well so not sure it would be wise to name it anything specific to security

Reply:
@cnasikas, @mikecote this connector will be a generic connector for Open AI integrations - Kibana wide. Observability team are planning to use it. And usage of that action connector functionality is unique - we don't want it to be a part of Security actions or general Rules actions. I believe this category makes sense.